src/components/PriceToggleJason/styles/NormalPackageCard.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components'
+
+export const NormalPackageContainer = styled.div<{ left: boolean }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 3vh;
+  height: 90%;
+  width: 280px;
+  background: hsl(0, 100%, 100%);
+  ${(props) => (props.left ? 'border-radius: 10px 0px 0px 10px;' : 'border-radius: 0px 10px 10px 0px;')}
```

Comment:
can take our border-radius since there will always be one; meaning `border-radius: ${.......`

Reply:
Resolved in c7d400b